tensorboard/plugins/audio/plugin_data.proto

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+syntax = "proto3";
+
+package tensorboard;
+
+// Audio summaries created by the `tensorboard.plugins.audio.summary`
+// module will include `SummaryMetadata` whose `plugin_data` field has
+// as `content` a binary string that is the encoding of an
+// `AudioPluginData` proto.
+message AudioPluginData {
+  enum Encoding {
+    // Do not use `UNKNOWN`; it is only present because it must be.
+    UNKNOWN = 0;
+    WAV = 11;
```

Comment:
That's fine. My argument would be that if I use the Python API, I don't care about how the data gets stored, I just care that it gets stored. But if I read a proto, then I'm very interested in how the data is structured.